.github/workflows/python-publish.yml

```diff
@@ -24,13 +24,13 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@v5
         with:
-          python-version: "3.9"
+          python-version: "3.12"
 
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          python -m pip install -e '.[dev]'
-      
+          python -m pip install --group=dev -e .      
+          
       - name: Test with pytest
         run: pytest
```

Comment:
Alternately, this could be done simply with `uv pytest`, right? Then we wouldn't need the pip command beforehand?

Reply:
Yes, it would also make sense to just convert everything to uv...